web/modules/custom/paragraph_align/paragraph_align.routing.yml

```diff
@@ -0,0 +1,7 @@
+paragraph_align:
+  path: '/paragraph/page'
+  defaults:
+    _controller: '\Drupal\paragraph_align\Controller\ParagraphController::para'
```

Comment:
para()?

Why so short method name? It should describe what method should to do.